src/main/java/org/opensearch/security/support/ConfigConstants.java

```diff
@@ -296,6 +298,8 @@ public enum RolesMappingResolution {
     public static final String TENANCY_GLOBAL_TENANT_NAME = "global";
     public static final String TENANCY_GLOBAL_TENANT_DEFAULT_NAME = "";
 
+    public static final String USE_JDK_SERIALIZATION = "plugins.security.use_jdk_serialization";
```

Comment:
Do you plan to set a default for this setting in `OpenSearchSecurityPlugin.getSettings()`?: https://github.com/opensearch-project/security/blob/main/src/main/java/org/opensearch/security/OpenSearchSecurityPlugin.java#L899-L1095

Reply:
> Thank you @parasjain1! I took a first pass and left comments with a few questions that I have. Can you outline the steps you took to do backwards compatibility testing?
> 
> What does this look like during a rolling upgrade?

I tested this by creating a cluster with OS2.5, kept some traffic flowing via OSB, and then triggered an upgrade to OS2.7 (the code was modified to have a check on OS2.5 instead of OS2.7 for testing). The upgrade succeeded and the cluster continued to operate normally. This signifies that the old and the new nodes were able to communicate properly. 

I've planned for a few additional rounds of testing especially with indices having DLS, FLS and Masked Field configurations.

Reply:
Please refer to this comment - https://github.com/opensearch-project/security/pull/2802#discussion_r1223179782